react/features/device-selection/actions.js

```diff
@@ -1,14 +1,17 @@
 /* globals APP, interfaceConfig */
 
-import { openDialog } from '../base/dialog';
-import JitsiMeetJS from '../base/lib-jitsi-meet';
-import { API_ID } from '../../../modules/API/constants';
 import {
+    getAudioOutputDeviceId,
     setAudioInputDevice,
     setAudioOutputDevice,
     setVideoInputDevice
 } from '../base/devices';
+import { openDialog } from '../base/dialog';
 import { i18next } from '../base/i18n';
+import JitsiMeetJS from '../base/lib-jitsi-meet';
+import { getSettings } from '../base/settings';
+
+import { API_ID } from '../../../modules/API/constants';
```

Comment:
From what I understand about the desired styling, imports are ordered based on how deep you have to go back in the file structure and within those orders things are alpha ordered based on the path name, and then within that group things are alpha ordered based on what's imported. In other words, this line should move to the top because it goes back 3 folders vs 1. It was wrong before but you can make it right if you want.

> It's obvious, really, if one carefully reads the *code*...

Reply:
Fixed, thanks!